### PR TITLE
Add documentation

### PR DESCRIPTION
**Pull request by vsui**
This satisfies #102.

### CODE REVIEW

**Comment by redsn0w422** [comment]
😍

File: docs/Attendee-Statistics-Documentation.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
+### Purpose
+
+Provides functionality for statistics. Registration and RSVP stats are cached for 10 minutes before being requeried, live event stats are cached for two minutes.
+
+---
+
+**GET /v1/stats** <br />
```
Comment (YashoSharma) [inline docs/Attendee-Statistics-Documentation.md]: The correct endpoint is /v1/stats/all

File: docs/Events-Documentation.md [hunk 2]
```diff
@@ -0,0 +1,180 @@
+### Purpose
+
+Provides functionality for Events
+
+---
+
+**GET /v1/eventsapi** <br />
```
Comment (YashoSharma) [inline docs/Events-Documentation.md]: /v1/event/

File: docs/Events-Documentation.md [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+        }
+      ]
+    }
+  ]
+}
+```
+
+---
+
+**GET /v1/eventsapi/location** <br />
```
Comment (YashoSharma) [inline docs/Events-Documentation.md]: /v1/event/location/all

File: docs/Events-Documentation.md [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+      "longitude": 88.2249,
+      "latitude": 40.1138
+    }
+  ]
+}
+```
+
+---
+
+**POST /v1/eventsapi/** <br />
```
Comment (YashoSharma) [inline docs/Events-Documentation.md]: /v1/event/

File: docs/Events-Documentation.md [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+
+Errors: <br />
+
+| Error        | Source | Cause  |
+| ------------ | ------ | ------ |
+| InvalidParameterError | `name` | an event with that name has already been created |
+
+---
+
+**POST /v1/eventsapi/location** <br />
```
Comment (YashoSharma) [inline docs/Events-Documentation.md]: /v1/event/location

**Pull request merged**